src/main/java/org/continuouspoker/dealer/persistence/daos/LogEntryDAO.java

```diff
@@ -0,0 +1,56 @@
+package org.continuouspoker.dealer.persistence.daos;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import io.quarkus.hibernate.orm.panache.PanacheQuery;
+import io.quarkus.panache.common.Sort;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.transaction.Transactional;
+import org.continuouspoker.dealer.LogEntry;
+import org.continuouspoker.dealer.persistence.entities.LogEntryBE;
+import org.continuouspoker.dealer.persistence.mappers.LogEntryMapper;
+import org.mapstruct.factory.Mappers;
+
+@ApplicationScoped
+public class LogEntryDAO {
+    private static final int GAME_LOG_LIMIT = 50;
```

Comment:
I'm not sure whether a hard coded limit makes sense at this level. I would probably leave it up to the caller, otherwise we might lose flexibility here.